packages/qwik/src/cli/new/run-new-command.ts

```diff
@@ -78,13 +84,19 @@ export async function runNewCommand(app: AppCommand) {
 
     if (typeArg === 'route') {
       outDir = join(app.rootDir, 'src', `routes`, nameArg);
+    } else if (typeArg === 'markdown') {
+      outDir = join(app.rootDir, 'src', `routes`, nameArg);
```

Comment:
can we combine these two branches?
```suggestion
    if (typeArg === 'route' || typeArg === 'markdown') {
      outDir = join(app.rootDir, 'src', `routes`, nameArg);
```